CMarkGFM.hsc

```diff
@@ -73,7 +72,7 @@ extsToLlist (h:t) = do
 
 -- | Resolves a CMarkExtension to its pointer.
 resolveExt :: CMarkExtension -> IO (Maybe ExtensionPtr)
-resolveExt e = do
+resolveExt e = registerPlugins `seq` do
```

Comment:
I started to think this might need to be `pseq`, but I'm pretty sure that's not the case given the second argument to `seq` is in IO, right?